### PR TITLE
[R] add ambiorix framework

### PR DESCRIPTION
**Pull request by JohnCoene**
Added the ambiorix framework (R language)

### CODE REVIEW

File: r/ambiorix/config.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+framework:
+  website: ambiorix.dev
+  version: 2.1.0
```
Comment (waghanza) [inline r/ambiorix/config.yaml:3]:
```suggestion
  version: 2.1
```

**Review by waghanza — CHANGES_REQUESTED** [review]
minor change about config file

**Comment by JohnCoene** [comment]
@waghanza I'm sorry, it's my own framework and I managed to mess this up.
Should be good for a re-run of github action

**Comment by waghanza** [comment]
@JohnCoene et voila ... https://web-frameworks-benchmark.netlify.app/result?f=ambiorix

**Comment by JohnCoene** [comment]
Thank you!

**Pull request merged**